### ISSUE TITLE
Suggest: Test Explorer exists by default, but off

### ISSUE THREAD

**Issue by chrisdias** [microsoft/vscode#225462]
The Test Explorer only shows up once you install an extension that contributes a Test provider.  Suggestion here is that there is a default Test Explorer view, but it is turned off by default. If you right click the Activity Bar, "Testing" is there by default but unchecked. When you turn it on and no Test provider extensions are installed, we could have a Welcome View that gives a brief overview of Testing and a link to query the Marketplace for Test Provider extensions.

**Comment by connor4312** [comment]
@isidorn we discussed this a little bit. I know different folks have some opinions here

**Comment by chrisdias** [comment]
just to clarify, I'm trying to find some middle ground with this suggestion. I don't think having it on by default is the right answer, but there is also so much goodness that is not obvious, so wondering if a "welcome" experience might help.

**Comment by isidorn** [comment]
Yeah, either it does not show up (as now), or it shows up with a welcome view (to guide users towards installing Testing Extensions).
Sounds like a good candidate for an experiment. Not sure if there are other ideas?
